client/disputes/evidence/fields.js

```diff
@@ -117,10 +118,29 @@ const sections = [
 	{
 		key: 'duplicate_charge_info',
 		title: __( 'Duplicate charge info', 'woocommerce-payments' ),
+		description: interpolateComponents( {
+			mixedString: __(
+				'Click to {{learnMoreLink}}learn more{{/learnMoreLink}}',
+				'woocommerce-payments'
+			),
+			components: {
+				learnMoreLink: (
+					// eslint-disable-next-line jsx-a11y/anchor-has-content
+					<a
+						rel="external noopener noreferrer"
+						target="_blank"
+						href="https://woo.com/document/woopayments/"
+					/>
+				),
+			},
+		} ),
```

Comment:
Getting these small tweaks wrapped up might take a little longer:

- Dependent on docs.
- Would benefit from some design input.

I'd suggest shifting changes to the dispute form to a separate PR. They don't block the other fix (showing charge id), and are less urgent. Also tough to get the right balance for a short-term fix here as we plan to revisit the form in a future project – making _any_ changes here might be pulling a thread.

Reply:
Good idea!

Reverting changes on evidence form page here cf5abe2456f182dfdd80287f94b0a576e20f2b1b.